tests/test_dotnet.py

```diff
@@ -16,13 +16,18 @@
 from bci_tester.data import DOTNET_RUNTIME_7_0_CONTAINER
 from bci_tester.data import DOTNET_SDK_6_0_CONTAINER
 from bci_tester.data import DOTNET_SDK_7_0_CONTAINER
+from bci_tester.data import OS_VERSION
 from bci_tester.util import get_repos_from_connection
 
 
 #: Name and alias of the microsoft .Net repository
 MS_REPO_NAME = "packages-microsoft-com-prod"
 
 
+@pytest.mark.skipif(
+    OS_VERSION == "tumbleweed",
+    reason="no Tumbleweed dotnet containers",
+)
```

Comment:
you can replace all these individual `skipif` with a single module wide declaration:
```Python
pytestmark = pytest.mark.skipif(
    OS_VERSION == "tumbleweed",
    reason="no Tumbleweed dotnet containers",
)
```

Reply:
thanks for the tip, done